lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/OperatorTopology.cs

```diff
@@ -392,7 +400,7 @@ private void WaitForTaskRegistration(IList<string> identifiers)
 
                 var leftOver = string.Join(",", identifiers.Where(e => !foundList.Contains(e)));
                 Logger.Log(Level.Error, "For node {0}, cannot find registered parent/children: {1}.", _selfId, leftOver);
-                Exceptions.Throw(new ReefRuntimeException("Failed to initialize operator topology for node: " + _selfId), Logger);
+                Exceptions.Throw(new RemotingException("Failed to find parent/children nodes in operator topology for node: " + _selfId), Logger);
```

Comment:
In IMRU, this method is called from task  where we catch bunch of exceptions. RemotingException is one of them and we treat it as communication exception. 
In other GC tasks examples/tests, we don't catch this exception so far.